tests/test_link.py

```diff
@@ -186,6 +186,17 @@ def test_title_as_init_argument(self) -> None:
         assert link.title == link_title
         assert link.to_dict().get("title") == link_title
 
+    def test_serialize_link(self) -> None:
+        href = "https://some-domain/path/to/item.json"
+        title = "A Test Link"
+        link = pystac.Link(pystac.RelType.SELF, href, pystac.MediaType.JSON, title)
+        link_dict = link.to_dict()
+
+        assert str(link_dict["rel"]) == "self"
```

Comment:
Is this project using unittest still, or has it migrated to pytest? Might want self.assertEqual.

Reply:
We're still on `unittest`. I would definitely be in favor of switching to `pytest` if someone were willing to do the legwork to convert our tests.

Reply:
Changed these to `self.assertEqual` in the latest commit, thanks for the suggestion!